release-notes/opensearch-dashboards-maps.release-notes-2.2.0.0.md

```diff
@@ -2,5 +2,20 @@
 Compatible with OpenSearch and OpenSearch Dashboards Version 2.2.0
 
 ### Features
+* Remove plugin check logic and support offline working of importing custom geoJSON([#10](https://github.com/opensearch-project/dashboards-maps/pull/10))
+* Add error modal and lint fixes([#13](https://github.com/opensearch-project/dashboards-maps/pull/13))
+* Introduce refresh option to update the custom vector map list ([#14](https://github.com/opensearch-project/dashboards-maps/pull/14))
 
-* Support custom GeoJson by uploading GeoJson files and visualize data in dashboard region maps ([OpenSearch#831](https://github.com/opensearch-project/OpenSearch/issues/831)).
+### Bug Fixes
+* Fix failing tests and updates license headers in files([#11](https://github.com/opensearch-project/dashboards-maps/pull/11))
+* Fix issue with file validation and danger toast messages ([#16](https://github.com/opensearch-project/dashboards-maps/pull/16))
```

Comment:
We don't need this, since it is part of dev ( pre release )

Reply:
updated